platform/linux-generic/odp_ipsec.c

```diff
@@ -1526,18 +1526,7 @@ static ipsec_sa_t *ipsec_out_single(odp_packet_t pkt,
 	}
 
 	if (!crypto.ok) {
```

Comment:
@JannePeltonen Should we make line#1522 as,

`if ((rc < 0) || (!crypto.ok))`

and remove the separate code block line#1528-line#1531?

Only difference we have is the ODP_DBG print.

Reply:
No, because crypto.ok check cannot be moved above the odp_crypto_result() call.